content/english/post/calhoun-minneapolis.md

```diff
@@ -15,10 +15,22 @@ Bde Maka Ska ("White Earth Lake") in Minneapolis, Minnesota was restored to its
 
 <!--more-->
 
-From at least 1839 through 2018, Bde Maka Ska bore the name "Lake Calhoun", in honor of [enthusiastic slaveholder and racist John C. Calhoun](), who as United States secretary of war had sent the army to survey the area.  
+From at least 1839 through 2018, Bde Maka Ska bore the name "Lake Calhoun", in honor of enthusiastic slaveholder and racist John C. Calhoun, who as United States secretary of war had sent the army to survey the area.[^1]
 
-In 2011, the Minneapolis Park and Recreation Board considered renaming the lake, but determined that they did not have the legal authority to do so.  In 2015, a petition was launched to change the name, culminating in name changes at the state and federal level in 2018. 
+In 2011, the Minneapolis Park and Recreation Board considered renaming the lake, but determined that they did not have the legal authority to do so.  In 2015, a petition was launched to change the name, culminating in name changes at the state and federal level in 2018.
 
-Restoring the name Bde Maka Ska was opposed by the group "[Save Lake Calhoun](http://www.savelakecalhoun.com)\*", who sued to prevent it.  The Minnesota court of appeals ruled in their favor, but the Minnesota supreme court [upheld the name change in May 2020](https://www.startribune.com/minnesota-dnr-can-rename-lake-calhoun-as-bde-maka-ska-high-court-rules/570435552/).  
+Restoring the name Bde Maka Ska was opposed by the group "Save Lake Calhoun\*", who sued to prevent it.[^2] The Minnesota court of appeals ruled in their favor, but the Minnesota supreme court upheld the name change in May 2020.[^3]
```

Comment:
```suggestion
Restoring the name Bde Maka Ska was opposed by the group _Save Lake Calhoun_, who sued to prevent it.[^2] The Minnesota court of appeals ruled in their favor, but the Minnesota supreme court upheld the name change in May 2020.[^3]
```